daemon/api.go

```diff
@@ -973,6 +993,10 @@ func snapsOp(c *Command, r *http.Request, user *auth.UserState) Response {
 		return BadRequest("cannot decode request body into snap instruction: %v", err)
 	}
 
+	if inst.hasOpts() {
```

Comment:
This feels very specific but painted under a generic name. It may have options (e.g. channel), but we don't accept a particular set of options which would not make multi-snap ops happy.

As such, I'd suggest either making it purpose specific. E.g.:
```
func (i *...) ensureMultiSnapSafe() error {
        if ... {
                return fmt.Errorf("multi-snap operations don't take options")
        }
}
```
or alternatively we might just inline the conditional logic here.